### DISCUSSION TITLE
Including standalone html and javascript in quarto site

### DISCUSSION THREAD

**Discussion by JosiahParry** [quarto-dev/quarto-cli#7797] [Q&A]
### Description

I have a standalone non-interactive dashboard that I created with {bslib} that I would like to host in my quarto site. I've placed it inside of my quarto site with a hyperlink to the index.html. However, when I render the project the associated `./libs/` directory is not included. Is there any way to specify to include those?

**Comment by JosiahParry** [comment]
Note to other: include the files in `resources:` if you want to link from it. This solved it for me.
```yaml
resources: 
  - html/lib
```

**Comment by JosiahParry** [comment]
Reopening since this does not seem to work with `freeze: true`. The culprit: https://github.com/R-ArcGIS/r-bridge-site/blob/main/location-services/tutorials/shiny-dash/index.qmd

**Comment by mcanouil** [comment]
![image](https://github.com/quarto-dev/quarto-cli/assets/8896044/340673ed-23e6-4318-bd91-c6c081b26003)

Sorry it's very confusing ...
What's the status?

Also, note that when asking question like this one, it's really a requirement to provide a small and fully reproducible example from the start.
